pom.xml

```diff
@@ -51,6 +51,7 @@
         <dep.slice.version>0.27</dep.slice.version>
         <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
         <dep.tempto.version>1.18</dep.tempto.version>
+        <dep.zookeeper.version>3.4.6</dep.zookeeper.version>
```

Comment:
This isn't needed since it's only used in one place. We only add properties when multiple dependencies need to share the same version.